lib/src/test/java/com/auth0/android/lock/views/AuthConfigTest.java

```diff
@@ -73,8 +76,11 @@ public void shouldHaveValidDefaultName() throws Exception {
     public void shouldHaveValidDefaultLogo() throws Exception {
         ReflectionHelpers.setStaticField(Build.VERSION.class, "SDK_INT", Build.VERSION_CODES.JELLY_BEAN);
         AuthConfig defaultConfig = new AuthConfig(connection, R.style.Lock_Theme);
-        final Drawable drawable = RuntimeEnvironment.application.getResources().getDrawable(R.drawable.com_auth0_lock_ic_social_facebook);
-        Assert.assertThat(defaultConfig.getLogo(RuntimeEnvironment.application), is(equalTo(drawable)));
+        Drawable drawable1 = RuntimeEnvironment.application.getResources().getDrawable(R.drawable.com_auth0_lock_ic_social_auth0);
+        Drawable drawable2 = defaultConfig.getLogo(RuntimeEnvironment.application);
+        int d1 = shadowOf(drawable1).getCreatedFromResId();
```

Comment:
This also changed to comply with the new robolectric behavior https://github.com/robolectric/robolectric/issues/4037